### ISSUE TITLE
Controllo mappatura di campi (per il sito del catalogo)

### ISSUE THREAD

**Issue by ICCD-MiBACT** [ICCD-MiBACT/ArCo#87]
Per assicurare il funzionamento del sito del catalogo, occorre verificare la presenza, per ciascuna tipologia di scheda delle informazioni essenziali. Vi allego a breve una tabella Excel su cui stiamo lavorando

**Comment by margherita-porena** [comment]
Certo, stiamo già verificando quali informazioni vadano ancora modellate, ma sicuramente se ci fornite una lista delle più urgenti possiamo lavorare da subito su queste.

**Comment by ICCD-MiBACT** [comment]
Ho inserito un documento su cui però stiamo lavorando. Lo aggiornerò nei prossimi giorni...
